static/app/components/replays/replayContext.tsx

```diff
@@ -373,7 +373,7 @@ export function Provider({
           lineWidth: 2,
           strokeStyle: theme.purple200,
         },
-        plugins: organization.features.includes('session-replay-enable-canvas')
```

Comment:
Should we be going around and removing this flag if it's old/unused now?

Reply:
This still exists, it's used for recording canvas! We decided to add a new feature flag for replaying canvas.